phase2/kubeadm/configure-vm-kubeadm-master.sh

```diff
@@ -59,4 +59,7 @@ featureGates:
 EOF
 fi
 
+# first migrate the configuration from `v1alpha1` to any newer version.
+# don't error if `config migrate` is not available as a command.
+kubeadm config migrate --old-config $KUBEADM_CONFIG_FILE --new-config $KUBEADM_CONFIG_FILE || true
```

Comment:
I think we should just upload the new config v1alpha2, and not call migrate imo.